### PR TITLE
Add DIRAC

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dirac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python =2.7
+    - pip
+  run:
+    - python =2.7
```
Comment (chrisburr) [inline recipes/dirac/meta.yaml]: I hope to simplify these at some point but it's not easy to know what this actually needs. I'll fix it in the feedstock once I figure out how to do a better job of testing.

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review.

Windows fails due to missing dependencies but it's `noarch` so it doesn't matter.

File: recipes/dirac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+{% set name = "DIRAC" %}
+{% set version = "6.21.5" %}
+{% set major, minor, patch = version.split(".") %}
+
+package:
+  name: {{ name|lower }}
```
Comment (isuruf) [inline recipes/dirac/meta.yaml]: There are lots of packages with the name dirac. What's the name that other package managers use for this package?

Comment (chrisburr) [reply recipes/dirac/meta.yaml]: That's a good point. It's not in any other package managers but I've renamed it to `dirac-grid` which matches the website URL and GitHub organisation.

File: recipes/dirac-grid/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    - matplotlib >=2.1.0,<3.0
+    - mysqlclient >=1.2.5
+    - jinja2
+    - numpy >=1.10.1
+    - pexpect >=4.0.1
+    - psutil >=4.2.0
+    - pyasn1 >0.4.1
+    - pyasn1-modules
+    - pytz >=2015.7
+    # - pyreadline >=6.2.4
```
Comment (scopatz) [inline recipes/dirac-grid/meta.yaml]: remove dead line?

Comment (chrisburr) [reply recipes/dirac-grid/meta.yaml]: Done 👍

**Pull request merged**